sonar-python-plugin/src/main/java/org/sonar/plugins/python/PythonScanner.java

```diff
@@ -343,19 +340,19 @@ private void saveMetricOnFile(InputFile inputFile, Metric<Integer> metric, Integ
   }
 
   private void handleQuickFixes(InputFile inputFile, RuleKey ruleKey, NewIssue newIssue, PreciseIssue preciseIssue) {
-    if (isInSonarLint(context) && newIssue instanceof NewSonarLintIssue) {
+    if (isInSonarLint(context)) {
```

Comment:
did you check everything works fine in SonarLint? I would suggest deploying the jar locally in VSCode and test quickfixes

Reply:
checked, looks good